polyfills/String/prototype/padEnd/polyfill.js

```diff
@@ -0,0 +1,17 @@
+(function () {
+	Object.defineProperty(String.prototype, 'padEnd', {
+		configurable: true,
+		enumerable: false,
+		value: function padEnd (targetLength, padString) {
```

Comment:
This may be a silly question/observation but I thought the `target` was normally the thing the method was being called on?
But actually as I write this comment out I realise that it actually makes sense that the target whould be the thing you are trying to achieve 🤦‍♂️ (I'm going to keep this comment as written proof that I do realise things)